schemas/caching-schema.json

```diff
@@ -24,6 +24,20 @@
                                             "type": "boolean",
                                             "description": "Enable debug logs in discovery service"
                                         },
+                                        "heap": {
+                                            "type": "object",
+                                            "description": "Configure JVM parameters for heap size",
+                                            "properties": {
+                                                "init": {
```

Comment:
JVM_HEAP_SIZE_INIT vs. JVM_HEAP_INIT

Reply:
as JVM_HEAP_SIZE_INIT is being dropped only ZWE_configs_heap_init remains valid